app/scrambler.py

```diff
@@ -0,0 +1,59 @@
+"""Scrambler module."""
+import random
+
+from picture import Picture
+from PIL import Image, ImageDraw
+
+
+class Scrambler:
+    """Scrambler class."""
+
+    def scramble_rows(self, picture: Picture) -> Picture:
+        """Split an Image up into rows an rearranges them."""
+        raise NotImplementedError
+
+    def scramble_grid(self, picture: Picture) -> Picture:
+        """Split an Image up into tiles an rearranges them."""
+        raise NotImplementedError
+
+    @classmethod
+    def scramble_circle(cls, picture: Picture, num_tiles: int = 6) -> Picture:
+        """Split an Image up into rings an rearranges them."""
+        image = picture.image
+        if image is not None:
+            width, height = image.size
+            center = (width // 2, height // 2)
+
+            radiusvar = min(width, height) / (2 * num_tiles)
+
+            result = Image.new("RGB", (width, height), (255, 255, 255))
```

Comment:
I would recommend using `image.mode` so this supports RGBA images. This caused an issue for me in the qualifier.

Reply:
I convert the image back to its original mode to not face that issue.